service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceRegisterTask.java

```diff
@@ -261,6 +263,21 @@ private boolean compareAndReRegisterSchema(Entry<String, String> localSchemaEntr
       String scSchemaContent = srClient.getSchema(microservice.getServiceId(), scSchema.getSchemaId());
       String localSchemaContent = localSchemaEntry.getValue();
 
+      if (isIllegalValue(scSchemaContent) && isIllegalValue(localSchemaContent)) {
+        Swagger scSwagger = SwaggerUtils.parseSwagger(scSchemaContent);
+        Swagger localSwagger = SwaggerUtils.parseSwagger(localSchemaContent);
+        if (scSwagger.equals(localSwagger)) {
+          if (ServiceRegistryConfig.INSTANCE.isIgnoreSwaggerDifferent()) {
+            LOGGER.warn(
+                "the local schema[{}]'s content is same with the service center schema's content, but the order of some "
+                    + " parameters in the configuration file is inconsistent:\n service center schema:\n[{}]\n local schema:\n[{}]",
+                localSchemaEntry.getKey(),
+                scSchemaContent,
+                localSchemaContent);
+          }
```

Comment:
This block should executed when not equals.

Reply:
done